restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/mariadb/init.sql

```diff
@@ -188,7 +188,8 @@ voice_method VARCHAR(4),
 voice_fallback_url MEDIUMTEXT,
 voice_fallback_method VARCHAR(4),
 voice_application_sid VARCHAR(34),
-uri MEDIUMTEXT NOT NULL
+uri MEDIUMTEXT NOT NULL,
+push_client_identity VARCHAR(34)
```

Comment:
First of all we need to add flag to DB that this RC Client uses mobile application. This flag defines how CallManager will handle SIP Invite. After a long discussion we decided to use Identity instead of just a flag.